src/main/java/org/ms/zk/plugin/matcher/HostMatcher.java

```diff
@@ -0,0 +1,15 @@
+package org.ms.zk.plugin.matcher;
+
+import com.unboundid.ldap.sdk.LDAPException;
+
+import java.io.IOException;
+
+public class HostMatcher extends Matcher {
+
+  @Override
+  public boolean matches(String id, String aclExpr) throws IOException, LDAPException {
+    return aclExpr.startsWith("host/")
+      && id.contains("@")
+      && aclExpr.equals(id.split("@", 2)[0]);
```

Comment:
we are ignoring realm of the host, generally speaking we should not, but can be changed later. i would prefer if somewhere in the config file we specify and load realm (or realm list). can be later.